packages/backend/src/accounting/tigerbeetle/service.ts

```diff
@@ -212,8 +212,15 @@ export async function getAccountBalance(
   deps: ServiceDependencies,
   id: string
 ): Promise<bigint | undefined> {
-  const account = (await getAccounts(deps, [id]))[0]
-  if (account) return calculateBalance(account)
+  const stopTimer = deps.telemetry.startTimer('tb_get_account_balances', {
```

Comment:
maybe we can also add one for the TB transfer creation? Might be handy

Reply:
Added. Also added it for PSQL.